src/ilastik-app.imjoy.html

```diff
@@ -64,7 +64,7 @@
     method: 'GET',
     responseType: 'arraybuffer',
     onDownloadProgress: (progressEvent) => {
-      showMessage(`Downloading weights ${Math.round(progressEvent.loaded/progressEvent.total*100)}% (${progressEvent.total/(1024*1024)} MB)`);
+      showMessage(`Downloading weights ${Math.round(progressEvent.loaded/progressEvent.total*100)}% (${(progressEvent.total/(1024*1024)).toFixed(2)} MB)`);
```

Comment:
Do you want to do the same with the percentage?

Reply:
I'd say for percentage it's fine... But I can do it if you'd rather have it somewhat consistent.

Reply:
No, I thought you would prefer a more precise estimation of the percentage ;)

Reply:
:)